class/practice2.1/main.cpp

```diff
@@ -0,0 +1,59 @@
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <unordered_map>
+
+void loadStatistics(std::unordered_map<std::string, int>& stats) {
+    std::ifstream infile("statistics.txt");
```

Comment:
use constants instead of hardcoding the magic lines.
namespace {
    constexpr std::string databasePath = "...
    // also put other strings here like command names bread, delete and so on
}